catalog/ESA/ESA_WorldCereal_2021_MODELS_v100.jsonnet

```diff
@@ -0,0 +1,133 @@
+    
+    **Table 1: WorldCereal seasons description**
+    | Season |Description|
+    | :---------------------- | :---------------------------------------- |
+    | tc-annual        | A one-year cycle being defined in an AEZ by the end of the last considered growing season  |
+    | tc-wintercereals | The main cereals season defined in an AEZ
+    | tc-springcereals | Optional springcereals season, only defined in certain AEZ
+    | tc-maize-main | The main maize season defined in an AEZ
+    | tc-maize-second | Optional second maize season, only defined in certain AEZ
+     
+    The available products in this collection are:
+    - **temporarycrops**
+    - **maize**
+    - **wintercereals**
+    - **springcereals**
+    - **irrigation**
+     
+    Each product (image) has a binary classification (0|100) and a confidence (0-100) band. Note that AEZs for which no irrigation product is available were not processed because of the unavailability of thermal Landsat data.
+    
+    The collection should be filtered using one or more of the following image properties:
+    - **aez_id**, holding the ID of the AEZ to which the image belongs
+    - **product**, describing the WorldCereal product name of the image
+    - **season**, describing for which season the image is valid.
+    
+    **References:**
+    [1] [WorldCereal methodology and products paper](https://doi.org/10.5194/essd-2023-184)
+    [2] Need to reference the AEZ FeatureCollection here.
+    [3] [WorldCereal global seasonality paper](https://doi.org/10.1080/15481603.2022.2079273)
+  |||,
+  license: license.id,
+
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference)],
+  keywords: ['agriculture', 'copernicus', 'crop', 'esa', 'global', 'landcover', 'landsat', 'sentinel1_derived', 'sentinel2_derived'],
+  providers: [
+    ee.producer_provider(
+      'ESA WorldCereal Consortium',
+      'https://esa-worldcereal.org/en'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2020-01-01T00:00:00Z', '2021-12-31T23:59:59Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'aez_id',
+        description: 'ID of the AEZ to which the product belongs.',
+        type: ee_const.var_type.int
+      },
+      {
+        name: 'product',
+        description: 'WorldCereal product name.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'season',
+        description: 'Season for which the product is valid.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'start_date',
```

Comment:
Actually these are the same. `start_date` and `end_date` properties of an image are just `YYYY-MM-DD` formatted versions of system start and end time of that same image. Might have been unclear during our discussion. If you think this is confusing, we can omit these properties.

Reply:
Let's omit them from the dataset description, and if they somehow become necessary, we'll add them back.

Some datasets set month and/or julian day as a separate property to speed up filtering on thoe attributes, but because crops have different cycles I suspect these won't be of much use here.

Reply:
Removed them in d9285ba8e1f416ba66e7e2d3df4068b691a7ff1f